ci/autoscaler/tasks/update-sdk/task.yml

```diff
@@ -14,10 +14,10 @@ inputs:
 
 params:
   GIT_USER_NAME: app-autoscaler-ci-bot
-  GIT_USER_EMAIL: InstanceManagerCloudNotifications@sap.com
+  GIT_USER_EMAIL: ApplicationAutoscaler@sap.com
   UPLOADER_KEY: ((autoscaler_blobstore_uploader_key))
   GITHUB_ACCESS_TOKEN: ((autoscaler_access_token))
-  GITHUB_PRIVATE_KEY: ((autoscaler_git_key))
```

Comment:
I replaced all deploy key usages with the new ssh authentication key of the `autoscaler-ci-bot`. 

Shall we completely get rid off the the deploy key here https://github.com/cloudfoundry/app-autoscaler-release/settings/keys and simply delete it? 🤷‍♂️

Reply:
Let's try it :)

Reply:
Yeah, worst case we can bring it back easily : ) The key still lives in CredHub